### PR TITLE
Updates page layout and style.

### PR DESCRIPTION
**Pull request by bsonntag**
Updates navbar and page content layout and style.

### CODE REVIEW

File: app/views/shared/_navbar.slim [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+    = link_to 'Botler', root_path, class: 'navbar-brand' 
 
-  = link_to 'Sign up', new_user_registration_path
+    ul.nav.navbar-nav.navbar-right
 
-  = link_to 'Sign in', new_user_session_path
+      - if user_signed_in?
+
+        li
+
```
Comment (ronaldofs) [inline app/views/shared/_navbar.slim]: just remove these extra spacings; minor writing style change :)

**Comment by ronaldofs** [comment]
:+1:

File: app/views/shared/_navbar.slim [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
-- else
+    = link_to 'Botler', root_path, class: 'navbar-brand' 
 
-  = link_to 'Sign up', new_user_registration_path
+    ul.nav.navbar-nav.navbar-right
 
-  = link_to 'Sign in', new_user_session_path
+      - if user_signed_in?
+
+        li = button_to 'Log out', destroy_user_session_path, method: 'delete', class: 'btn btn-link navbar-btn'
```
Comment (bsonntag) [inline app/views/shared/_navbar.slim:11]: @ronaldofs Like this?

**Comment by ronaldofs** [comment]
go! :shipit:

**Pull request merged**